### ISSUE TITLE
[Feature] OHLCV-Based Intraday Entry Plan System

### ISSUE THREAD

**Issue by iAmGiG** [iAmGiG/AutoTrader-AgentEdge#366]
## Problem

The current entry system uses **simple market price + fixed offsets** without leveraging intraday OHLCV (Open, High, Low, Close, Volume) data for intelligent entry planning. This results in:

1. **Missed optimal entries**: No pullback or dip detection
2. **No support/resistance awareness**: Doesn't use High/Low levels
3. **Volume ignored**: No confirmation of price moves
4. **Daily-only analysis**: VoterAgent uses 60 days of daily data, no intraday scanning
5. **Fixed risk levels**: -2% stop and +3.5% target regardless of volatility

### Current Entry Logic (Simplified)

```python
# From RealVoterStrategy and AlpacaExecutionManager
entry_price = current_market_price  # Latest quote/trade
stop_loss = entry_price * 0.98     # -2% fixed
take_profit = entry_price * 1.035   # +3.5% fixed

# Place market order at current price
place_bracket_order(entry, stop, target)
```

**Files Involved:**
- src/strategies/real_voter_strategy.py:140-146 - Fixed offset calculations
- src/execution/alpaca_execution_manager.py:143-151 - Market price execution
- src/autogen_agents/voter_agent.py:152-165 - Daily Close prices only

---

## Proposed Solution: OHLCV-Based Entry Plan System

Add a new **EntryPlanner** component that uses intraday OHLCV data to determine:
- **When to enter**: Pullback levels, breakouts, support/resistance
- **Where to enter**: Limit order prices based on OHLCV levels
- **How much to risk**: Dynamic stops based on ATR
- **Volume confirmation**: Validate entries with volume analysis

### What OHLCV Data Provides

**1. Support/Resistance from High/Low**
- Find recent swing lows (support) for pullback entries
- Find recent swing highs (resistance) for breakout entries
- Enter near support for better prices

**2. ATR-Based Dynamic Stops**
- Calculate Average True Range for volatility-based stops
- Wider stops for volatile stocks, tighter for stable stocks
- Adjust to market conditions automatically

**3. Volume Confirmation**
- Confirm entries with above-average volume
- Avoid low-volume price moves
- Validate breakouts with volume spikes

**4. Multiple Timeframe Analysis**
- Check alignment across 5min/15min/1hour
- Enter only when timeframes agree
- Reduce false signals

---

### Current Flow
```
VoterAgent (daily) → BUY/SELL → RealVoterStrategy (fixed offsets) → Market order
```

### Enhanced Flow
```
VoterAgent (daily) → BUY/SELL → OHLCVEntryPlanner (intraday) → Smart entry plan → Limit order
```

---

### Phase 1: Core Entry Planner
- [ ] Create src/trading_tools/entry_planner.py
- [ ] Implement support/resistance from High/Low
- [ ] Add ATR calculation for dynamic stops
- [ ] Add volume confirmation logic
- [ ] Unit tests

### Phase 2: Intraday Data
- [ ] Extend VoterAgent for multiple timeframes
- [ ] Add timeframe parameter to TradeRequest
- [ ] Update cache for intraday data
- [ ] Test with Polygon/Alpaca

### Phase 3: Integration
- [ ] Integrate with RealVoterStrategy
- [ ] Update AlpacaExecutionManager for limit orders
- [ ] Add CLI visualization
- [ ] User approval flow

### Phase 4: Advanced
- [ ] Multi-timeframe alignment
- [ ] Pullback detection
- [ ] Breakout detection
- [ ] Entry scaling
- [ ] Time-based restrictions

---

## Expected Benefits

1. **Better Entry Prices**: Enter at support vs random market price
2. **Dynamic Risk Management**: ATR-based stops adjust to volatility
3. **Volume Confirmation**: Avoid low-volume traps
4. **Pullback Support**: "Buy QQQ at a pullback" becomes actionable
5. **Multiple Timeframes**: 5min/15min/1hour confirmation
6. **Improved Win Rate**: Better entries = better risk:reward

---

### Before (Fixed Offsets)
```
QQQ: BUY
Entry: $505.23 (market)
Stop: $495.12 (-2.0%)
Target: $523.41 (+3.5%)
R:R: 1:1.74
```

### After (OHLCV-Based)
```
QQQ: BUY

OHLCV Entry Plan (15min):
  Support: $502.15
  Resistance: $508.50
  ATR: $2.34

Entry: $502.65 (pullback to support)
Stop: $499.14 (1.5 ATR)
Target: $509.67 (2:1 R:R)

✅ Volume 1.3x average
✅ 5m/15m/1h aligned
💡 Limit order at $502.65
```

---

## Related Issues

- #344 - Pullback/timing context parsing
- #361 - LLM intent classification
- #310 - AutoGen agent completion

## Priority

**High** - Major improvement using existing infrastructure

## References

- VoterAgent: voter_agent.py:152-165 (daily Close only)
- Fixed offsets: real_voter_strategy.py:140-146
- Market execution: alpaca_execution_manager.py:143-151
- OHLCV support: unified_market_tool.py:41-90

**Comment by iAmGiG** [comment]
Implemented in commit 3bf530c on `feature/core-execution` branch.

**Changes:**
- New `entry_planning.py` module with ATR, support/resistance, and volume confirmation functions
- `calculate_entry_plan()` generates dynamic entry/exit levels using OHLCV data
- RealVoterStrategy now uses OHLCV-based entry planning with fallback to fixed percentages
- Plan quality scoring (STRONG/MODERATE/WEAK) based on volume and price position

**Features:**
- ATR-based stop-loss (2x ATR multiplier)
- Support/resistance-aware stop adjustment
- 2:1 risk/reward ratio targeting
- Volume confirmation for entry validation
